src/phaser-game/scenes/SingleMode.ts

```diff
@@ -23,7 +20,7 @@ let collidableObjects:
   | Phaser.GameObjects.GameObject[]
   | Phaser.GameObjects.Group
   | Phaser.GameObjects.Group[];
-let shootBullet: (x: number, y: number, direction: string) => void;
+let shootBullet: any;
```

Comment:
i'll allow it, if you can't do better than 'any' here

Reply:
I tried my best to remove type any from the phaser game, but there are certain circumstances where even when I wrote the Phaser type or type it would still throw an error that was only solved by using type any.